commons/src/main/java/org/infinispan/commons/marshall/UTF8Marshaller.java

```diff
@@ -0,0 +1,29 @@
+package org.infinispan.commons.marshall;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+
+import org.infinispan.commons.io.ByteBuffer;
+import org.infinispan.commons.io.ByteBufferImpl;
+
+public final class UTF8Marshaller extends AbstractMarshaller {
```

Comment:
I prefer to have multiple instances of marshaller, for each charset, since it needs to used in the server xml, and for clarity, having a ```marshaller="UTF8Marshaller"``` is more descriptive than ```marshaller="StringMarshaller"```. What if someone needs a different charset?